src/Definition/ClassDefinition.php

```diff
@@ -130,6 +130,20 @@ public function compile(Context $context)
         $context->setFilepath($this->filepath);
         $context->setScope($this);
 
+        foreach ($this->properties as $property) {
+            if ($property->default == null) {
```

Comment:
Please don't write strict compare, when is not needed, why `if ($property->default` is not okey for you?